src/Hazelcast.Net.Tests/Remote/ClientTxnNonSmartTest.cs

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System.Linq;
```

Comment:
```suggestion
```

Reply:
Don't worry too much about these. There will be another PR that essentially clears all these rogue using (it's done with an automatic tool).